moped-etl/arcgis/components_to_agol.py

```diff
@@ -220,7 +282,9 @@ def main(args):
                 logger.info(f"Deleting features with project ids {joined_project_ids}")
                 delete_features_by_project_ids(feature_type, joined_project_ids)
 
-            features = all_features[feature_type]
+            features_of_type = all_features[feature_type]
+
+            features = handle_status_updates(features_of_type)
```

Comment:
i like this new function—you might want to move it up before `delete_features_by_project_ids`. that way, if it happens to error out you won't be left with partially deleted features.

Reply:
i'm updating this so that incremental mode has the same order of function calls in the full mode. thanks for flagging! 🙏